mergin/merginproject.py

```diff
@@ -731,3 +731,15 @@ def resolve_unfinished_pull(self, user_name):
         shutil.rmtree(self.unfinished_pull_dir)
         self.log.info("unfinished pull resolved successfuly!")
         return conflicts
+
+    def set_tables_to_skip(self, tables):
+        """
+        Set list of tables to exclude from geodiff operations. Once defined, these
+        tables will be excluded from the following operations: create changeset,
+        apply changeset, rebase, get database schema, dump database contents, copy
+        database between different drivers.
+
+        Tables passes as semicolon separated list, e.g. "table1;table2;...;tableN".
+        If empty string is passed, list will be reset.
+        """
+        self.geodiff.set_tables_to_skip(tables)
```

Comment:
is this then valid for all gpkg files, right?

Don't we need to bump geodiff version?

Reply:
Right, it will be applied to all geopackages.

As for the geodiff version we will need to bump it once geodiff is released.